rules/when-to-use-domain-and-integration-events/rule.md

```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know when to use Domain and Integration Events?
+guid: e91f22d2-d385-49b6-a27b-1195b4bb13f9
+uri: when-to-use-domain-and-integration-events
+created: 2023-07-25T11:01:35.0000000Z
+authors:
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
+archivedreason:
+related: []
+redirects:
+---
+
+Domain Events and Integration Events are powerful patterns used in Domain-Driven Design (DDD) to enable loosely coupled communication between different parts of an application. They serve as a means of **notification** for important domain concepts that have occurred.
+
+<!--endintro-->
+
+## What are Domain Events
+
+In the Domain Layer of the [Clean Architecture](https://github.com/sswconsulting/SSW.CleanArchitecture), Domain Events are an integral part of the Domain Model.
```

Comment:
I think this could be a little confusing or even misleading. It's integral to SSW's Clean Architecture template (and solutions), but it's not part of Clean Architecture, as an architectural pattern.

Reply:
I'll make it clear that we are talking about CA + DDD to strengthen this idea.